scratch/db_connection_strings/connection_string_timeouts.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
The scratch dir is destined for the wastebin. Can this go to the ge_notebooks repo?

Reply:
Added a task for thin in 0.8.5